deployment_scripts/appimage/README.md

```diff
@@ -0,0 +1,37 @@
+# Monkey Island AppImage
+
+## About
+
+This directory contains the necessary artifacts for building a prototype
+monkey_island AppImage using appimage-builder.
+
+## Building an AppImage
+
+1. Create a clean VM or LXC (not docker!) based on Ubuntu 18.04.
+1. Update the OS with `sudo apt update && sudo apt upgrade`
+1. Copy the `deployment_scripts/appimage` directory to `$HOME/` in the VM.
+1. Run `sudo -v`.
+1. On the VM, `cd $HOME/appimage`
+1. Execute `./build_appimage.sh`. This will pull all necessary dependencies
+   and build the AppImage.
+
+NOTE: This script is intended to be run from a clean VM. You can also manually
```

Comment:
Woulnd't it be easier in terms of integration testing, to NOT require a new machine each time?

Reply:
LXCs are very quick to spin up. So are VMs in the right environment. Regardless, the README contains instructions on how to clean up after the script runs so that you don't need to use a fresh environment each time.